src/preview-app.html

```diff
@@ -190,7 +190,9 @@
     },
     attached: function() {
       // On page-load, we may or may not be signed in.
-      this.$.firebase.checkSignedIn();
+      if (this.page.page !== '') {
```

Comment:
This doesn't work, we can not get off the homepage.
So, we we click login we are redirected to the homepage (etc.).